### PR TITLE
Exception Page showing only method names in the callstack

### LINKED ISSUE #304: Exception Page showing only method names in the callstack
image

Using:
dotnet 1.0.0-preview2-003024
Microsoft.Extensions.Diagnostics 1.0.0-rc3-21343

### PR DESCRIPTION
**Pull request by pranavkm**
Fixes #304

### CODE REVIEW

**Comment by pranavkm** [comment]
@rynowak \ @halter73 

image

**Comment by pranavkm** [comment]
Ping

File: src/Microsoft.AspNetCore.Diagnostics/DeveloperExceptionPage/Views/StackFrameInfo.cs [hunk 1]
```diff
@@ -1,54 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
```
Comment (BrennanConroy) [inline src/Microsoft.AspNetCore.Diagnostics/DeveloperExceptionPage/Views/StackFrameInfo.cs]: Are you allowed to remove this? Its public so "breaking change"

Comment (pranavkm) [inline src/Microsoft.AspNetCore.Diagnostics/DeveloperExceptionPage/Views/StackFrameInfo.cs]: cc @Eilon

Comment (halter73) [inline src/Microsoft.AspNetCore.Diagnostics/DeveloperExceptionPage/Views/StackFrameInfo.cs]: Ugh. With all the hacks, erm... workarounds, we're going to have to do to maintain binary compat, keeping a class around that no one will ever use (or indeed *has* ever used looking at the git history) is pretty minor I guess.

Presumably our tools for checking binary compat will yell at us later if we remove it now.

@Eilon Do we want to file issues for all the easy-to-make breaking changes we want to make in the next major release? Should we create a tag for this?

Comment (Eilon) [inline src/Microsoft.AspNetCore.Diagnostics/DeveloperExceptionPage/Views/StackFrameInfo.cs]: I'll work with folks to come up with official guidance on this. We definitely can't just remove this. But we can perhaps both document it as being obsolete, as well as mark it as `[Obsolete]` (*warning*, not *error*), to alert users. And then file a bug for 2.0.0 to delete it (we could actually go create that milestone if needed).

Comment (pranavkm) [inline src/Microsoft.AspNetCore.Diagnostics/DeveloperExceptionPage/Views/StackFrameInfo.cs]: I'll leave this type in. We should've put all the view generation code in an internal namespace - none of it is very useful.

**Comment by pranavkm** [comment]
:up: :date:

File: src/Microsoft.AspNetCore.Diagnostics/DeveloperExceptionPage/Views/StackFrameInfo.cs (hunk 1 above)
Comment (pranavkm) [inline src/Microsoft.AspNetCore.Diagnostics/DeveloperExceptionPage/Views/StackFrameInfo.cs]: I'll do the `Obsolete` attribute in a follow up. It's not particularly related to this change.

**Comment by Eilon** [comment]
:shipit:

**Pull request closed without merging**